cmd/ttt/ttt.go

```diff
@@ -0,0 +1,31 @@
+package ttt
```

Comment:
Please rename this package

Reply:
This is just test sample, so I remove it